pig/resources/org/partiql/pig/templates/kotlin-inspecting-visitor.ftl

```diff
@@ -0,0 +1,81 @@
+[#ftl output_format="plainText"]
```

Comment:
I simply extracted this into its own `.ftl` file, otherwise it remains unchanged.